person.rb

```diff
@@ -1,4 +1,7 @@
 class Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rental
```

Comment:
- [ ] Let's ensure a method for adding _rentals_ is present, so as to establish a a _rental_ `belong-to` _person_ relationship, like the code example below:.

```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .

  def add_rental(book, date)
    Rental.new(date, book, self)
  end

  .
  .
end
```